### DISCUSSION TITLE
Please help by pointing me to the theme css section

### DISCUSSION THREAD

**Discussion by Kilzzz** [elementary/wingpanel-indicator-session#168] [Q&A]
Hi, I create Manjaro spins. I want to add Pantheon to the desktop list, but I am running into a theming problem. I have searched, and searched, and searched, but have not found any documentation to the css file for the pantheon themes. I am not that good at css, but with a little help I may be able to solve this. I have worked on the .css that comes with the desktop and I have been able to change it. Here is an image of the area
![good-theme](https://user-images.githubusercontent.com/37912441/186294710-911ae261-feeb-43ed-9102-9e51ab31c674.jpeg)

But I would like to add a different theme, and there is where the problem lies.
![my-theme1](https://user-images.githubusercontent.com/37912441/186295053-def60454-bd20-4eb1-81c3-ab600d3302b0.jpeg)
It also happens in the notification drop down.
![my-theme2](https://user-images.githubusercontent.com/37912441/186295176-78077ee6-1555-4797-8cb4-db89c27c5742.jpeg)

I have a feeling they are both in the same area of the css, but I could be wrong. If someone could point me to the part of the css that controlls the area that is a white box it would help a lot.

Thanks

Jim

**Comment by danirabbit** [comment]
My guess is that this is because it's a listbox element and your stylesheet probably has hardcoded lists to have a white background, even in popovers. But it's hard to know without seeing your stylesheet.

You can reference the elementary stylesheet here: https://github.com/elementary/stylesheet

You can use Gtk Inspector to see what the CSS selectors are for certain elements and to test changes. Make sure the inspector keybinding is enabled for gtk3 in gsettings at `/org/gtk/settings/debug/enable-inspector-keybinding` and then use the keyboard shortcut `Ctrl` + `Shift` + `I`
